frontend/src/lib-customizations/common.tsx

```diff
@@ -62,7 +62,6 @@ export const absenceColors = {
   FORCE_MAJEURE: status.danger,
   FREE_ABSENCE: accents.a7mint,
   TEMPORARY_RELOCATION: status.warning,
-  TEMPORARY_VISITOR: status.warning,
```

Comment:
One instance is still present here https://github.com/espoon-voltti/evaka/blob/3b294a9a85664df5f514cf53e410687c5e6be6a9/frontend/src/lib-customizations/espoo/employee/assets/i18n/fi.ts#L2160